src/model/evaluator.py

```diff
@@ -0,0 +1,17 @@
+"""
+Base class for an Evaluator
+"""
+
+from abc import ABC
+
+
+class Evaluator(ABC):
+    """
+    Base class for an Evaluator
+    """
+
+    def evaluate(self, input_file_path):
```

Comment:
if it evaluatos on a specific detector we should tell the method which he should use no?
```suggestion
    def evaluate(self, input_file_path: str, detector: str):
```